websites/docs-smoke-test/src/code-examples/example.cs

```diff
@@ -0,0 +1,5 @@
+public class MyClass {
+  public static void main(String[] args) {
+    System.out.println("Hello World");
+  }
+}
```

Comment:
Are you sure this is correct C# code? 🧐

Reply:
I'm not sure, I found it somewhere online. I don't remember where.

Reply:
O I see, it's Java.